### PR TITLE
CI-2558 build term importer

### PR DESCRIPTION
**Pull request by CodeProKid**

### CODE REVIEW

File: php/cli.php [hunk 1]
```diff
@@ -0,0 +1,161 @@
+<?php
+
+WP_CLI::add_command( 'dfm-migration', 'DFM_Migration_CLI' );
+
+class DFM_Migration_CLI extends WP_CLI {
+
+	/**
+	 * Imports a list of terms from a scv file
```
Comment (keg) [inline php/cli.php]: type? CSV?

**Review by keg — CHANGES_REQUESTED** [review]
Just the one typo. Otherwise this looks really good, glad we're getting this in to the migration repo!

File: php/cli.php (hunk 1 above)
Comment (CodeProKid) [reply php/cli.php]: ay yeah, crap.

**Comment by CodeProKid** [comment]
@keg updated

**Pull request merged**